Integrations/integration-HybridAnalysis.yml

```diff
@@ -354,230 +431,639 @@ script:
-          type: string
-        - contextPath: HybridAnalysis.Search.verdict
-          description: Verdict of search result
-          type: string
-        - contextPath: HybridAnalysis.Search.environmentDescription
-          description: Environment description of search result
-          type: string
-        - contextPath: HybridAnalysis.Search.submitname
-          description: Submission name of search result
-          type: string
-        - contextPath: HybridAnalysis.Search.vxfamily
-          description: Family of search result
-          type: string
-        - contextPath: HybridAnalysis.Search.threatscore
-          description: Threat score of search result
-          type: string
-        - contextPath: HybridAnalysis.Search.type_short
-          description: type of search result (url, host, etc.)
-          type: string
-        - contextPath: HybridAnalysis.Search.size
-          description: Size of search result
-          type: number
-        - contextPath: File.Malicious.Vendor
-          description: or malicious files, the vendor that made the decision
-          type: string
-        - contextPath: File.Malicious.Description
-          description: For malicious files, the reason for the vendor to make the decision
-          type: string
-      description: Search the database using the Hybrid Analysis search syntax
-    - name: hybrid-analysis-detonate-file
-      arguments:
-        - name: entryId
-          required: true
-          description: War-room entry ID of sample file
-        - name: environmentID
-          description: environment ID to submit file to(get all via vx-get-environments).
-            Default is 100, or other WINDOWS ID
-        - name: delay
-          description: Delay wait time between calls (in seconds)
-          defaultValue: "3"
-        - name: timeout
-          description: Total wait time (in seconds)
-          defaultValue: "60"
-      description: Detonate file through Hybrid Analysis
-    - name: hybrid-analysis-get-report-status
-      arguments: []
-      outputs:
-        - contextPath: HybridAnalysis.Submit.State
-          description: The state of the process
-          type: string
-        - contextPath: HybridAnalysis.Submit.SHA256
-          description: The submission SHA256
-          type: string
-        - contextPath: HybridAnalysis.Submit.JobID
-          description: The JobID of the submission
-          type: string
-        - contextPath: HybridAnalysis.Submit.EnvironmentID
-          description: The environmentID of the submission
-          type: string
-      description: Get the state of the file submission
+  - name: hybrid-analysis-scan
+    arguments:
+    - name: file
+      required: true
+      description: The MD5, SHA1, or SHA256 hash of the file.
+    - name: malicious_threat_levels
+      default: true
+      description: Threat level values, which will determine if the file is malicious.
```

Comment:
@reutshal is it a good enough explanation?

Reply:
Better to talk with Alex about this one

Reply:
Changing this to :
A CSV list of threat level values. Files that have a threat level specified in the list will be considered malicious.